docs/getting-started/using-isis-first-steps/introduction-to-isis.md

```diff
@@ -0,0 +1,373 @@
+one space.
+
+Command line arguments are broken into two categories: reserved and
+parameter. Reserved arguments always begin with a dash (-) and change
+major things about how the program runs while parameter arguments supply
+data to the application and control how it processes that input data to
+create output data.
+
+
+## Reserved Command Line Arguments
+
+-----
+
+Reserved arguments all begin with a dash (-), and are not sensitive to
+case (i.e., -help is the same as -HElp). A list of basic reserved
+arguments includes:
+
+  - **-webhelp** Launch a web browser showing the ISIS help page for
+    that program. All other arguments will be ignored.
+  - **-help** Display a list of the program's parameters showing their
+    default values. For example:
+```
+    > equalizer -help
+    FROMLIST    = Null
+    HOLDLIST    = Null
+    TOLIST      = Null
+    OUTSTATS    = NULL
+    INSTATS     = NULL
+    PROCESS     = (*BOTH, CALCULATE, APPLY)
+    SOLVEMETHOD = (QRD, *SPARSE)
+    ADJUST      = (*BOTH, BRIGHTNESS, CONTRAST, GAIN)
+    MINCOUNT    = 1000
+    WEIGHT      = FALSE
+    PERCENT     = 100.0
+```
+
+  - **-last** Run the program using the same parameter arguments from
+    the most recent time the program was run. This does not include any
+    reserved arguments.
+  - **-restore=filename** Run the program using the arguments from the
+    file specified in filename.
+
+Reserved arguments come in two forms, solitary keyword or keyword value
+pair. Solitary keywords tell the program to perform some special action
+such as the -help keyword above. Keyword value pairs also tell the
+program to perform some special action, but use additional information
+for that action like in the -restore argument above. The keyword and
+value must be separated from each other with an equals sign (=). The
+equals sign may have spaces on either or both sides of it.
+
+## Parameter Command Line Arguments
+
+-----
+
+Parameter arguments are always made up of keyword value pairs. The
+keyword and value are separated by an equals sign just like reserved
+arguments, but they begin with an alphabetic letter instead of a dash.
+Parameters control what input data to use, what settings to use to
+process that input data, and where to put the output data. There are six
+different types of parameters:
+
+  - **Cube file names** - A cube file name is the location on disk of an
+    ISIS cube. It can be either the input to this program or where to
+    place the output. Input cubes, must be the output from some other
+    ISIS program. Most programs can process cubes from ISIS2, but some
+    information may be interpreted differently. Output cubes, will
+    always be the location to place processed results from the
+    application. When entering the filename of a cube it is not
+    necessary to enter the .cub extension. It will be added for you if
+    you do not type it.
+
+    ```
+    from=r0700563_lev1.cub
+    to= f431a62
```

Comment:
Is the space after `to=` intentional or should that be removed?

Reply:
I think those arguments are allowed with or without spaces

Reply:
Regardless, it's good to be consistent.